### ISSUE TITLE
Analiza sentymentu przez modele z rodziny BERT

### ISSUE THREAD

**Issue by mrutkowsky** [mrutkowsky/NAWROS#4]
Idea is to see how model behave on source data provided from SAMSUNG
- try using transfer learning with pretrained model

Trzeba zdjąć z gotowca softmax i dodać sigmoidę- żeby złagodniał

1. Labele
2. Modele obecne i na czym były przetrenowane
3. a)Trenujemy nowy model od 0 i patrzymy na wyniki 
    b)bierzemy wytrenowany model i próbujemy go dotrenować
    c) bierzemy dataset zwiazany z naszym problemem https://www.kaggle.com/datasets/lakshmi25npathi/imdb-dataset-of-50k- 
     movie-reviews z tego negatywne review + nasze zgłoszenia i na tym trenujemy
     Dla naszych zgłoszeń możemy dać inną wagę (większą).
